### PR TITLE
[task/ceph_ansible]: collect logs when the test fails

### PR DESCRIPTION
**Pull request by vasukulkarni**
Signed-off-by: Vasu Kulkarni <vasu@redhat.com>

### CODE REVIEW

**Comment by vasukulkarni** [comment]
http://pulpito.ceph.com/vasu-2017-06-22_21:30:50-rgw-master-distro-basic-vps/

**Comment by vasukulkarni** [comment]
retest this please

**Comment by vasukulkarni** [comment]
retest this please

File: teuthology/task/ceph_ansible.py [hunk 1]
```diff
@@ -253,6 +255,43 @@ def teardown(self):
...
+                        '-name',
+                        '*.log',
+                        '-print0',
+                        run.Raw('|'),
+                        'sudo',
+                        'xargs',
+                        '-0',
+                        '--no-run-if-empty',
+                        '--',
+                        'gzip',
```
Comment (zmc) [inline teuthology/task/ceph_ansible.py]: ``misc.pull_directory()`` already gzips, doesn't it?

Comment (vasukulkarni) [inline teuthology/task/ceph_ansible.py]: I see it runs tar there not gzip and this command command gzips files locally, also this command is verbatim from ceph.py.

Comment (dmick) [inline teuthology/task/ceph_ansible.py]: @zmc: yes, it does

Comment (vasukulkarni) [inline teuthology/task/ceph_ansible.py]: I see the mode='r|gz' does gzip so i can remove this command but needs to be tested again.

**Pull request merged**